rpcserver.go

```diff
@@ -174,6 +174,10 @@ var (
 			Entity: "onchain",
 			Action: "write",
 		}},
+		"/lnrpc.Lightning/BumpFee": {{
+			Entity: "onchain",
+			Action: "write",
+		}},
```

Comment:
We are gradually working towards moving rpc calls into sub servers and split up `lnd`. Should we still add something new to the main rpc server?

Reply:
I wouldn't be opposed to doing so - seems like this should apply to the `pendingsweeps` RPC as well? Would we want to add these to `WalletKit` or have a new `Sweeper` sub-server?

Reply:
I would think `WalletKit`. @Roasbeef probably has an idea about it too

Reply:
Moved to `WalletKit` after offline discussion.